Fastfile.private

```diff
@@ -283,6 +274,28 @@ reflected in `Info.plist` during building."
 
   end
 
+  desc "Updates or downloads the `Certificates` and `Provisioning Profiles`."
+  private_lane :refresh_certificates do |options|
+
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    match_type = get_match_type(build_configuration: build_configuration_key)
+ 
+    desc "Bundle identifier from xCode project"
+    bundle_identifier = read_project_property(
+      build_configuration: build_configuration,
+      build_setting: 'PRODUCT_BUNDLE_IDENTIFIER'
+    )
+
+    desc "Download provisioning profile and certificates if not present."
```

Comment:
Where are you checking if they are present or not to create or download?

Reply:
Oh, okay. so it will fail with a message the user will understand right?

Reply:
Let me check the message. Not sure what it is, since it's a fastlane one. I'll be back to you with this.

Reply:
It says:

```
No matching provisioning profiles found for 'AppStore_ar.com.Wolox.BaseProject'
A new one cannot be created because you enabled `readonly`
Provisioning profiles in your repo for type `appstore`:
......
If you are certain that a profile should exist, double-check the recent changes to your match repository
```

Reply:
@danielaRiesgo I think it's clear enough 👍

Reply:
Great ! It would be so nice to add "Use command create_..." but it's okay if not :)